docs/compatibility.md

```
@@ -147,23 +146,40 @@ Replacement for JetBrains `StringUtil` naming:
 
 - Text with `{property}` holes
 - `{{` / `}}` escapes
 - `@` destructure and `$` stringify
-- alignment and format
-- property names: letters, digits, `_`, `.`, space
-- malformed holes become text
+- Alignment with an optional leading `-` followed by one or more digits,
+  including zero and widths larger than `Int32.MaxValue`
+- A non-empty format containing any character except `}`, including extra `:`
+  or `,` (`{Timestamp:HH:mm:ss}`, `{Value:#,0}`). `{Bad: {Good}}` is one hole
+  whose format is ` {Good`, not a nested property.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)